### ISSUE TITLE
Add MacOS support

### ISSUE THREAD

**Issue by kgraefe** [kgraefe/pidgin-master-password#7]

**Comment by kgraefe** [comment]
environment:
```
export PATH="/usr/local/opt/gettext/bin:$PATH"
export LDFLAGS="-L/usr/local/opt/gettext/lib"
export CPPFLAGS="-I/usr/local/opt/gettext/include"
export ACLOCAL_PATH="/usr/local/opt/gettext/share/aclocal:$ACLOCAL_PATH"
export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig:$PKG_CONFIG_PATH"
```

**Comment by kgraefe** [comment]
Build Deps:
intltool automake libtool pkg-config

Deps:
pidgin libsodium

**Comment by kgraefe** [comment]
Resolved by 0b24123
